net/nfs-kernel-server/Makefile

```diff
@@ -166,12 +188,13 @@ HOST_CONFIGURE_VARS += \
 	GSSGLUE_LIBS=" " \
 	RPCSECGSS_CFLAGS=" " \
 	RPCSECGSS_LIBS=" " \
-	CONFIG_SQLITE3_TRUE="\#" \
-	CONFIG_NFSDCLD_TRUE="\#" \
 	ac_cv_lib_event_core_event_base_dispatch=yes \
 	ac_cv_header_event2_event_h=yes \
 	c_cv_lib_sqlite3_sqlite3_libversion_number=yes \
-	libsqlite3_cv_is_recent=yes
+	libsqlite3_cv_is_recent=yes \
+	ac_cv_header_libdevmapper_h=yes \
+	ac_cv_header_sys_inotify_h=yes \
```

Comment:
This line, `ac_cv_header_sys_inotify_h=yes \` is not required. I took them all from confugure.ac, but then checked confirmed that only libdevmapper ones were necessary.